tests/std/tests/GH_003570_allocate_at_least/test.cpp

```diff
@@ -51,7 +51,7 @@ struct signalling_allocator {
 
     allocation_result<T*> allocate_at_least(size_t count) {
         allocate_at_least_signal.set();
-        return {allocate(count * 2), count * 2};
+        return {allocate(count * 2 + 1), count * 2 + 1};

```

Comment:
`count * 2` happens to work; the deque will break easily if users try to implement their own `allocate_at_least`.